src/System.Transactions.Local/src/System/Transactions/TransactionOptions.cs

```diff
@@ -23,7 +23,7 @@ public IsolationLevel IsolationLevel
 
         public override int GetHashCode() => base.GetHashCode();  // Don't have anything better to do.
 
-        public override bool Equals(object obj) => obj is TransactionOptions && Equals((TransactionOptions)obj);
+        public override bool Equals(object? obj) => obj is TransactionOptions && Equals((TransactionOptions)obj);
```

Comment:
Can remove the cast if you use the already in use `is` operator to declare inline:

```suggestion
        public override bool Equals(object? obj) => obj is TransactionOptions transactionOptions && Equals(obj);
```